ts/features/idpay/wallet/components/IDPayInitiativesListComponents.tsx

```diff
@@ -64,18 +69,15 @@ export const IDPayInitiativeListItem = ({
   const { initiativeName } = item;
 
   return (
-    <NBlistItem
-      itemDivider={false}
-      last={isLast}
-      style={{
-        justifyContent: "space-between",
-        paddingRight: 0,
-        paddingLeft: 0 // required to override NBlistItem default padding
-      }}
-    >
-      <H6 color="black">{initiativeName}</H6>
-      <SwitchOrStatusLabel item={item} idWallet={idWallet} />
-    </NBlistItem>
+    <View>
+      <View style={styles.listItemContainer}>
+        <View style={styles.leftContent}>
+          <NewH6 numberOfLines={1}>{initiativeName}</NewH6>
+        </View>
+        <SwitchOrStatusLabel item={item} idWallet={idWallet} />
```

Comment:
This component uses `RemoteSwitch`, which still depends on Native-Base.
We need to use `NativeSwitch`.

Reply:
It's not possible to replace the `RemoteSwitch` with the `NativeSwitch` directly since the first one is a wrapper that handles different statuses of the `pot`. What should be done in my opinion is to replace inside the `RemoteSwitch` component the deprecated `NBSwitch` with the new `NativeSwitch` but I think it is better to address that in another PR, what do you think?

Reply:
> It's not possible to replace the `RemoteSwitch` with the `NativeSwitch` directly since the first one is a wrapper that handles different statuses of the `pot`. What should be done in my opinion is to replace inside the `RemoteSwitch` component the deprecated `NBSwitch` with the new `NativeSwitch` but I think it is better to address that in another PR, what do you think?

This is the way imo, we should probably see if the RemoteSwitch is still in the designSystem's scope and in case add a new task. @dmnplb what do you think?

Reply:
@Hantex9 @forrest57 @mastro993  We should use the `SwitchListItem` instead of recreating the component using `RemoteSwitch`. If `NativeSwitch` (or its container `SwitchListItem`) needs some fixes and/or improvements, they should be added via a PR to the `io-app-design-system` repository.

If we don't, we add a non-standard variation of the component, which adds to the UI clutter.

Reply:
@dmnplb Unfortunately the `SwitchListItem` doesn't handle the possibility to show a `Badge` component on his right side as showed in the following screenshot (the second item in the list) that actually is used as a feedback message showing the status at the user since it is an async process (it means when a user tap on the switch and the request is successful, the status will not change quickly but in a second time)

<img src="https://github.com/pagopa/io-app/assets/34343582/91dcae34-7f59-4a84-ac67-cd9fbd48876e" width="200" />

Reply:
@Hantex9 @dmnplb maybe a listItem that takes a custom component in the right side ?

Reply:
After discussing in async about that with @thisisjp, it was decided for this PR to change the copy of the `Badge` with the text "In attesa.." and a `warning` variant as indicated also in the ticket [IOBP-184](https://pagopa.atlassian.net/browse/IOBP-184) and addressed in b2143069131cc9d4d4bb0261f5d7f2dd5a3d4b41

About the creation of another ListItem component i think it is better to evaluate and discuss in another thread or in the weekly sync.

@dmnplb @mastro993 @forrest57

Reply:
@Hantex9 @forrest57 Yes, but it should be named with a precise meaning (not generic `rightComponent`), otherwise `SwitchListItem` could potentially be used as a generic `ListItem` component

Reply:
> @Hantex9 @dmnplb maybe a listItem that takes a custom component in the right side ?

@dmnplb @Hantex9 @forrest57  We should create a base ListItem which has everything but the right element, which can be configured via props.